README.md

```diff
@@ -79,6 +79,8 @@ SONiC
 
 # Controller
 
+- [lighty.io core](https://github.com/PantheonTechnologies/lighty-core) - lighty.io core components - An open source development framework for building Java-based SDN controllers.
```

Comment:
Duplicated to lighty.io

Reply:
lighty-core is a subset of lighty.io .. 
lighty-core = core functionality 
lighty.io = lighty-core + proprietary improvements done by Pantheon.tech